src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -3,20 +3,49 @@ pragma solidity 0.8.13;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./BountyQuestion.sol";
+
+// Interfaces
 import "./interfaces/IClaimController.sol";
 import "./interfaces/IQuestionStateController.sol";
 import "./interfaces/IActionCostController.sol";
+
+// Modifiers
 import "./modifiers/NFTLocked.sol";
 
-// TODO a lot of talk about "admins" -> solve that
+/**
+ * @title MetricsDAO question API
+ * @author MetricsDAO team
+ * @notice This contract is an API for MetricsDAO that allows for interacting with questions & challenges.
+ */
+
 contract QuestionAPI is Ownable, NFTLocked {
     BountyQuestion private _question;
     IQuestionStateController private _questionStateController;
     IClaimController private _claimController;
     IActionCostController private _costController;
 
+    /// @notice Keeps track of the id of the most recent question created.
```

Comment:
what's the difference between @notice and @dev?

Reply:
From what I saw @notice is more general (solidity style docs: Explains to an end user what it does), where dev is more technical/extra detail